src/CairoMakie.jl

```diff
@@ -372,17 +389,19 @@ function to_cairo_image(img::AbstractMatrix{<: AbstractFloat}, attributes)
 end
 
 function to_cairo_image(img::Matrix{UInt32}, attributes)
-    CairoARGBSurface([img[j, i] for i in size(img, 2):-1:1, j in 1:size(img, 1)])
+    CairoARGBSurface(
+        [
+            img[j, i]
+            for i in size(img, 2):-1:1, # account for Y-axis discrepancy in Cairo
+                j in 1:size(img, 1)
+        ]
+    )
```

Comment:
is this the same as `collect(img')`? Maybe document more what's going on here. Probably the column major, row major conversion?

Reply:
This is just flipping the y-axis, it needs to be transposed for conversion to a surface.  I didn't change the code - just formatted it :D.   Will add docs.